server/server.js

```diff
@@ -269,9 +269,14 @@ APP.post('/api/auth/logout', (req, res) => {
 // Handles any requests that don't match the ones above.
 // All other routing except paths defined above is done by React in the UI
 APP.get('*', async(req,res) =>{
-    var campaign, title, description, image, url;
+    var title = "HEO App";
+    var description = "Crowdfunding on blockchain.";
+    var image = "https://app.heo.finance/static/media/heo-logo.e772bc1b.png";
+    var url = "https://app.heo.finance";
+    var campaign;
     var splitURL = req.url.split('/'); 
     var campaignId = splitURL[splitURL.length -1]
```

Comment:
I think this line will throw an error if there are no "/" in req.url and splitURL is an empty array, because then it will be trying to access minus first element of an empty array. I'd move this inside `if(splitURL.length > 2)`

Reply:
i've tested it a bunch of times and there is always a / in url. even an empty one gives me [ '', '' ] that array. http://, it catches those two. i've tested it on local host and local tunnels. it's always array of two empties at the minimum. Should i still rewrite or leave it?

Reply:
ok, sounds good